eng/pipelines/live-test-cleanup-template.yml

```diff
@@ -0,0 +1,25 @@
+parameters:
+  - name: DisplayName
+    type: string
+  - name: SubscriptionConfigurations
+    type: object
+    default:
+      - $(sub-config-azure-cloud-test-resources)
+
+steps:
+  - template: /eng/common/TestResources/build-test-resource-config.yml
+    parameters:
+      SubscriptionConfigurations: ${{ parameters.SubscriptionConfigurations }}
+
+  - pwsh: |
+      $subscriptionConfiguration = @'
+        $(SubscriptionConfiguration)
+      '@ | ConvertFrom-Json -AsHashtable
+
+      ./eng/scripts/live-test-resource-cleanup.ps1 `
+      @subscriptionConfiguration `
+      -WhatIf `
```

Comment:
I assume this is still here for testing?

Reply:
Yep, I've since moved testing into a separate branch. Will clean this up before merge.